GoogleDataTransport/GDTCORLibrary/Public/GDTCORPrioritizer.h

```diff
@@ -16,10 +16,11 @@
 
 #import <Foundation/Foundation.h>
 
+#import <GoogleDataTransport/GDTCOREvent.h>
 #import <GoogleDataTransport/GDTCORLifecycle.h>
 #import <GoogleDataTransport/GDTCORUploadPackage.h>
 
-@class GDTCORStoredEvent;
+@class GDTCOREvent;
```

Comment:
You imported GDTCOREvent--you don't need to forward declare it because you've imported it. I'd do one or the other. Something to think about: do you think adopters of this protocol should have to import the GDTCOREvent header, or should we just do it for them in this header? My opinion: headers that declare protocols should import the headers of classes required by the API so the adopting class only needs to import a single header--but maybe that's not such a good idea.